pyproject.toml

```diff
@@ -23,8 +23,8 @@ dependencies = [
     "photutils >=1.13.0",
     "pyparsing >=2.4.7",
     "requests >=2.26",
-    # "roman_datamodels>=0.22.0,<0.23.0",
-    "roman_datamodels @ git+https://github.com/spacetelescope/roman_datamodels.git",
+    #"roman_datamodels>=0.22.0,<0.23.0",
+    "roman_datamodels @ git+https://github.com/braingram/roman_datamodels.git@source_catalog",
```

Comment:
Temporarily pointing to the source branch for https://github.com/spacetelescope/roman_datamodels/pull/428 to pick up required changes.